swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, fabric="Unknown", condition=0.0):
+        super().__init__(id)
+        self.fabric = fabric 
+        self.condition = condition
```

Comment:
Here and in the other derived classes, you should take full advantage of the parent class's initializer and forward `condition`:

```python
        super().__init__(id = id, condition = condition)
        self.fabric = fabric 
```